onnxruntime/core/providers/webnn/builders/impl/pad_op_builder.cc

```diff
@@ -116,11 +121,42 @@ Status PadOpBuilder::AddToModelBuilderImpl(ModelBuilder& model_builder,
     options.set("value", value);
```

Comment:
Typo above.

`"WebNN dose not support mode"` ->
`"WebNN does not support mode"`